VelodyneHDL/IO/Lidar/Velodyne/vtkVelodynePacketInterpreter.cxx

```diff
@@ -1222,12 +1221,15 @@ void vtkVelodynePacketInterpreter::ComputeCorrectedValues(const unsigned short a
 
     if (insideAbsValue > 0)
     {
-      computedIntensity = computedIntensity + correction->focalSlope * insideAbsValue;
+      computedIntensity = computedIntensity + 100*correction->focalSlope * insideAbsValue;
     }
     else
     {
-      computedIntensity = computedIntensity + correction->closeSlope * insideAbsValue;
+      computedIntensity = computedIntensity + 100*correction->closeSlope * insideAbsValue;
```

Comment:
These should be made into a separate upstream pull request, as it's a suspected bug (as opposed to a brand-new feature).